.bazelrc

```diff
@@ -41,8 +41,12 @@ build:asan --define signal_trace=disabled
 build:asan --define ENVOY_CONFIG_ASAN=1
 build:asan --copt -fsanitize=address,undefined
 build:asan --linkopt -fsanitize=address,undefined
+# TODO(lizan): vptr and function requires C++ UBSAN runtime which we're not currently linking to.
+# Enable them when bazel has better support for that.
```

Comment:
This seems a pretty major loss of ASAN coverage; is there a reason this only shows up in clang-9? Can you also file a tracking issue?

Reply:
Because function ubsan added dependency to RTTI (which requires C++ runtime) https://reviews.llvm.org/D61479.